.env.release.example

```
@@ -1,8 +1,7 @@
-APPLE_DEVELOPER_ID_APP_P12_PATH="$HOME/certs/developer-id-application.p12"
-APPLE_DEVELOPER_ID_APP_P12_PASSWORD="replace-me"
-APPLE_DEVELOPER_ID_APP_PROFILE_PATH="$HOME/certs/MFuse-App.provisionprofile"
-APPLE_DEVELOPER_ID_EXTENSION_PROFILE_PATH="$HOME/certs/MFuse-Provider.provisionprofile"
 APPLE_TEAM_ID="TEAMID1234"
-APPLE_NOTARY_KEY_ID="ABC123DEF4"
-APPLE_NOTARY_ISSUER_ID="00000000-0000-0000-0000-000000000000"
-APPLE_NOTARY_API_KEY_PATH="$HOME/certs/AuthKey_ABC123DEF4.p8"
+APPLE_NOTARY_KEYCHAIN_PROFILE="mfuse-notary"
+MFUSE_BASE_VERSION="1.0"
+SIGNING_IDENTITY="Developer ID Application"
+PROFILES_DIR="$HOME/Library/Developer/Xcode/UserData/Provisioning Profiles"
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The PROFILES_DIR key should go before the SIGNING_IDENTITY key

(UnorderedKey)